apps/webapp/src/components/shared/input-token.tsx

```diff
@@ -98,19 +70,11 @@ export default function InputToken({
       </div>
 
       <div className='mt-[6px] flex items-center justify-between gap-2'>
-        <div className='flex items-start gap-2'>
-          {feeAsString && (
-            <>
-              <img src='/fuel.svg' alt='Gas fee' className='size-5' />
-              <p className='font-bold text-muted-foreground'>{feeAsString}</p>
-            </>
-          )}
-        </div>
```

Comment:
We're no longer rendering the fee inside `InputToken`, but rather as its own first-class field.